checker/src/options.rs

```diff
@@ -86,6 +86,8 @@ impl Options {
     /// Parses options from a list of strings. Any content beyond the leftmost `--` token
     /// will be returned (excluding this token).
     pub fn parse(&mut self, args: &[String]) -> Vec<String> {
+        self.single_func = std::env::var("MIRAI_SINGLE_FUNC").ok();
```

Comment:
Can't use MIRAI_FLAGS=--single_func=name or similar? I think this was the substitute when it was removed (based on your feedback to not add a ton of env vars.)

Reply:
Ah, good point. I've been a bit too set in my ways.